apps/src/applab/applab.js

```diff
@@ -824,18 +836,25 @@ async function initDataTab(levelOptions) {
         if (!datasetInfo) {
           // We don't know what this table is, we should just skip it.
           console.warn(`unknown table ${table}`);
-        } else if (datasetInfo.current) {
-          Applab.storage.addCurrentTableToProject(
-            table,
-            () => console.log('success'),
-            outputError
-          );
         } else {
-          Applab.storage.copyStaticTable(
-            table,
-            () => console.log('success'),
-            outputError
-          );
+          // TODO: post-firebase-cleanup, remove this conditional when we're done with firebase: #56994
+          if (isFirebaseStorage()) {
+            if (datasetInfo.current) {
+              Applab.storage.addCurrentTableToProject(
+                table,
+                () => console.log('success'),
+                outputError
+              );
+            } else {
+              Applab.storage.copyStaticTable(
+                table,
+                () => console.log('success'),
+                outputError
+              );
+            }
+          } else {
+            Applab.storage.addSharedTable(table);
+          }
```

Comment:
Because DatablockStorage tables are de-duplicated by default, there's no distinction between a "current" table and a static table. Static tables are also "current", the only difference is if there's a cron job that periodically updates the static table.